app/components/signin-code.js

```diff
@@ -0,0 +1,19 @@
+import Ember from 'ember';
+
+const { Component } = Ember;
+
+export default Component.extend({
+  didInsertElement() {
+    this._super(...arguments);
+
+    let $ = this.$.bind(this);
```

Comment:
This is some Javascript trickery that sets up a local function called `$` that is really the component's `this.$` function, so it will act just like jQuery's `$`, but only match elements contained in the component. This is what allows you to write the code (below) just like you're using regular jQuery, but it will only run when we want it to and only match the elements we want it to.